### DISCUSSION TITLE
avoidSnapshotBatteryDrain not working

### DISCUSSION THREAD

**Discussion by WBB85** [dgreif/ring#1100] [General]
Hi. First of all thanks for this amazing plugin.

I have several Ring cameras and 2 of them are on battery only. I enabled avoidSnapshotBatteryDrain so that the batteries are not drained by the snapshot every 10 seconds. But my problem is that the snapshot is still updated every 10 seconds on my battery cameras all though I have avoidSnapshotBatteryDrain enabled.

I use the Home app as dashboard on my wall mounted iPad, so the app is always running and the cameras are always visible in the Home app.

Did I do anything wrong or is it a bug?

**Comment by dgreif** [comment]
There was indeed a bug introduced recently. I've released a fix in `11.7.0-beta.3` and will cut a real release in the next few days

**Reply by WBB85** [reply]
Still the same problem here after updating to latest version. avoidSnapshotBatteryDrain is on, but battery camera snapshots are still updated every 15 seconds

**Reply by dgreif** [reply]
@WBB85 it may show to update in the Home app, but the important details is whether those are actually _new_ snapshots or being pulled from cache. Should be able to look at your homebridge logs to see how the snapshots are pulled.
